packages/graphql-cli-codegen/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@graphql-cli/codegen",
```

Comment:
Maybe just change the directory to be `graphql-cli-codegen-plugin` and `cli` ? it's a bit confusing now :D

Reply:
sure

Reply:
@dotansimha renamed